observability/WFLY-20296-add-support-for-more-opentelemetry-config-options.adoc

```diff
@@ -0,0 +1,140 @@
+---
+categories:
+- observability
+stability-level: community
+# Specify the Feature Development tracker issue for the feature.
+# This must be an issue tracked in https://github.com/orgs/wildfly/projects/7/views/1.
+# To create a Feature Development tracker issue, go to  https://github.com/wildfly/wildfly-proposals/issues/new/choose
+# and select 'Feature Development'
+issue: https://github.com/wildfly/wildfly-proposals/issues/683
+feature-team:
+ developer: jasondlee
+ sme:
+  -
+ outside-perspective:
+  -
+promotes:
+promoted-by:
+---
+= WFLY-20296 - Add support for more OpenTelemetry configuration options
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+While every aspect of OpenTelemetry is technically configurable via MP Telemetry and MP Config, there have been several requests for more direct support for the various options in the management model. The RFE will identify and implement the options most likely to be of general interest to WildFly users.
+
+=== User Stories
+
+- As an application developer, I would like greater control in how OpenTelemetry is configured in the server without having to add an external library (e.g., MicroProfile Config) to my deployment(s).
```

Comment:
I'm not sure how to phrase this better, but a user would not add MicroProfile Config to their deployments as a library; i.e. by packaging a jar.

Reply:
Is that better?